src/com/epam/task/module4/theSimplestClassesAndObjects/task7/Point.java

```diff
@@ -0,0 +1,41 @@
+package com.epam.task.module4.theSimplestClassesAndObjects.task7;
+
+import static java.lang.Math.pow;
+import static java.lang.Math.sqrt;
+
+public class Point {
+    private Double x;
+    private Double y;
+
+    public Point(Double x, Double y) {
+        this.x = x;
+        this.y = y;
+    }
+
+    public Double getX() {
+        return x;
+    }
+
+    public void setX(Double x) {
+        this.x = x;
+    }
+
+    public Double getY() {
+        return y;
+    }
+
+    public void setY(Double y) {
+        this.y = y;
+    }
+
+    public static Double distance(Point point1, Point point2) {
```

Comment:
If you want to use service logic like here, it is better to use a different class for that purpose. You can move this method outside of the class and leave it in 'Main' class.

Reply:
I would suggest extracting methods distance and medianCross to the Main class. And rename them to 'getDistance' and 'getMedianCross', instead of 'get' verb, 'calculate' or some other can be used, just noun is inappropriate.